lib/commands/screenshot.js

```diff
@@ -0,0 +1,11 @@
+let commands = {};
```

Comment:
Why is this file necessary? Can we not just proxy _all_ `screenshot` requests to WDA?